test/service/RewardProgramService.test.ts

```diff
@@ -66,7 +66,7 @@ describe('RewardProgramService', () => {
         const transaction = transactions[0] as TransferTransaction;
         expect(transaction.type).eq(TransactionType.TRANSFER);
         expect(transaction.message.payload).eq(
-            'enrol C9767496987222790518114049299DD52114BF2A8F7E5F4B70BB2B6365FAFD34 https://fboucquez-agent-symbollocal.ngrok.io',
+            'enroll C9767496987222790518114049299DD52114BF2A8F7E5F4B70BB2B6365FAFD34 https://fboucquez-agent-symbollocal.ngrok.io',
```

Comment:
Message should be in this format => `enroll <agent_url> <base64_agent_ca_csr_file>`

base64_agent_ca_csr_file => when decoded the text should include `-----BEGIN CERTIFICATE REQUEST---xxx -----END CERTIFICATE REQUEST-----`

Reply:
Updated